front_end/entrypoints/rn_inspector/rn_inspector.ts

```diff
@@ -22,6 +22,7 @@ import type * as InspectorBackend from '../../core/protocol_client/InspectorBack
 
 Host.RNPerfMetrics.registerPerfMetricsGlobalPostMessageHandler();
 
+Host.rnPerfMetrics.setLaunchId(Root.Runtime.Runtime.queryParam('launchId'));
```

Comment:
Different casing here compared to L23? 🤔

Reply:
Yeah one is static/constructor. One is the global instance. Following the pattern of the upstream `UserMetrics`:

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/9370b35e1979ec6953ff59ed78c8751e4f463750/front_end/core/host/host-legacy.ts#L40-L48